CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt

```diff
@@ -1,15 +1,59 @@
 == 3030-12-31 Getting Started
 <p>If you're new to CONSTELLATION, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.introduction">introduction</a>.</p>
 
+== 2020-07-08 Rendering Labels Bug Fixed
+<p>A bug with labels being rendered with unreadable text after opening views has been fixed.</p>
+
+== 2020-07-07 Manage Templates Layout Improvements
+<p>The File &gt; Manage Templates window's layout has been improved.</p>
+
+== 2020-07-07 Import from Delimited Fixes
+<p>A number of improvements have been made to the Import from Delimited:</p>
+<ul>
+<li>The filtering options from the preview table have been fixed so that doing <strong>Row !=1</strong> works again.</li>
+<li>A UX improvement has been made to allow you to import files of one type per session. After importing the first file, the <strong>Import File Parser</strong> is grayed out</li>
+</ul>
+
+== 2020-06-18 Info Map Bug Fixed
+<p>A bug preventing running the Clustering %gt; Info Map plugins has been fixed.</p>
+
+== 2020-06-09 Select Top N Bug Fixed
+<p>A bug preventing running the <strong>Select Top N</strong> from the Data Access View has been fixed.</p>
+
+== 2020-05-21 Schema View
+<p>The Node and Transaction type tabs in the Schema View are now searchable.</p>
+
+== 2020-05-21 Hierarchical Clustering Improvements
+<p>A bug preventing the running of the hierachical clustering sometimes has been fixed.</p>
```

Comment:
running of the hierachical clustering <strong>plugin</strong> sometimes

Reply:
Thanks, updated.